cookbooks/convection-box/tutorial-onset-of-convection/tutorial_main.ipynb

```diff
@@ -0,0 +1,111 @@
+{
+ "cells": [
```

Comment:
why is this file not in the notebooks folder?

Reply:
True. The philosophy here is to put the "obvious" starting point at the top level. Agnostic about this. what do others think?